spyder/locale/zh_CN/LC_MESSAGES/spyder.po

```diff
@@ -2422,8 +2422,8 @@ msgid ""
 "please select <b>%s</b> to put graphics inside the console or <b>%s</b> to "
 "interact with them (through zooming and panning) in a separate window."
 msgstr ""
-"确定如何在控制台中显示图形。 如果不确定，请选择<b>%s</b>将图形放入控制台或<b>"
-"％s</b>以在单独的窗口中与它们进行交互（如缩放和平移）。"
+"确定如何在控制台中显示图形。 如果不确定，请选择<b>%s</b>将图形放入控制台或<b>％s</b>"
+"以在单独的窗口中与它们进行交互（如缩放和平移）。"
```

Comment:
@steff456, I found the error. The real reason for this is that the symbol used by the translator on the second replacement (`％`) is not a regular percentage symbol (`%`) (I don't know why).

Therefore, please leave this string as it was and simply change that symbol. After doing that, you'll see more errors like this one to appear too, so please fix them too.

Something I forgot to tell you is that you need to use a translation program (like `poedit`) to update the translation (i.e. to generate a new `.mo` file) after making changes to source translation file (i.e. the `.po` file).

So in this PR you need to submit both the new `po` and mo files.

Reply:
In other words, you need to open this file in Poedit, make your changes and save it. That will automatically generate a new `mo` file for you.